app/res/layout/screen_login.xml

```diff
@@ -22,9 +22,11 @@
     </LinearLayout>
 
     <LinearLayout
+        android:id="@+id/main_pane"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:layout_alignParentTop="true"
+        android:paddingTop="55sp"

```

Comment:
I honestly don't know why I had that there at all...may have been a left-over from something that seemed necessary pre- new UI. Good catch, sorry about that.